### ISSUE TITLE
Keros Error: Required ARIA children role not present for NavToggler

### ISSUE THREAD

**Issue by nikita0905** [OfficeDev/office-ui-fabric-react#8315]
### Environment Information 
 
- __Package version(s)__: @uifabric/dashboard: 0.42.1

### Please provide a reproduction of the bug in a codepen:
 All active elements are assigned role as menu and no children have been defined.

 
#### Actual behavior:
 
ARIA required children role is not found.
 
#### Expected behavior:
 
ARIA required children should be correctly defined for elements. 

![Screenshot (41)](https://user-images.githubusercontent.com/11412027/54346683-8054d900-466b-11e9-8e6c-07588d7720ef.png)

**Comment by KevinTCoughlin** [comment]
@Raghurk can you please triage the above `@uifabric/dashboard` accessibility issue as area owner

**Comment by nikita0905** [comment]
@KevinTCoughlin Could you share an update on this?

**Comment by nikita0905** [comment]
Could someone share an update on this?

**Comment by ecraig12345** [comment]
Closing this because the dashboard package has moved to a new location.
